phalcon/mvc/model/query.zep

```diff
@@ -2513,6 +2522,93 @@ class Query implements QueryInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * gets the read connection from the model if there no transaction inside the query object
```

Comment:
but i wrote them ! they are new and all mine :) ! I would only change the visibility of:

getTransactionConnection / getWriteConnection / getReadConnection

the ones I wrote who are currently not merged. but i can leave them public if you want I just wanted encapsulate them because they are never used outside of the class.